### ISSUE TITLE
Excpeption in using Tensorflow 1.12 from java on windows 10

### ISSUE THREAD

**Issue by cogmeta** [tensorflow/tensorflow#26296]
<em>Please make sure that this is a bug. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:bug_template</em>

**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow):
No
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):
Windows10
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device:
- TensorFlow installed from (source or binary):
jar
- TensorFlow version (use command below):
1.12.0
- Python version:
- Bazel version (if compiling from source):
- GCC/Compiler version (if compiling from source):
- CUDA/cuDNN version:
- GPU model and memory:

I am trying to use tensorflow from java program on windows 10. I have followed all the instruction from here https://www.tensorflow.org/install/lang_java carefully. I am getting following exception:

Mar 03, 2019 5:45:53 PM org.openqa.selenium.remote.ProtocolHandshake createSession
INFO: Detected dialect: OSS
Exception in thread "Thread-4" java.lang.UnsatisfiedLinkError: C:\Users\olelo\AppData\Local\Temp\tensorflow_native_libraries-1551615368173-0\tensorflow_jni.dll: A dynamic link library (DLL) initialization routine failed
	at java.lang.ClassLoader$NativeLibrary.load(Native Method)
	at java.lang.ClassLoader.loadLibrary0(Unknown Source)
	at java.lang.ClassLoader.loadLibrary(Unknown Source)
	at java.lang.Runtime.load0(Unknown Source)
	at java.lang.System.load(Unknown Source)
	at org.tensorflow.NativeLibrary.load(NativeLibrary.java:101)
	at org.tensorflow.TensorFlow.init(TensorFlow.java:66)
	at org.tensorflow.TensorFlow.<clinit>(TensorFlow.java:70)
	at org.tensorflow.Graph.<clinit>(Graph.java:361)

	

**Describe the current behavior**

**Describe the expected behavior**

**Code to reproduce the issue**
Provide a reproducible test case that is the bare minimum necessary to generate the problem.

**Other info / logs**
Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached.

**Comment by Ayush517** [comment]
I think this might be an error caused because of MS Visual C++ 2015 Redistributable package. Try obtaining msvcp140.dll and placing it into a folder in PATH. Maybe that helps.
There is a similar closed issue : #14456

**Comment by cogmeta** [comment]
I tried that. C++ 2015 Redistributable package is installed and msvcp140.dll  is on the path. It does nto resolve the issue. I have gone through every issue filed about tensorflow and windows. Nothing seems to resolve it.

**Comment by Ayush517** [comment]
You are installing it for CPU Support Only or GPU Support?

**Comment by cogmeta** [comment]
just cpu. Following instructions from here https://www.tensorflow.org/install/lang_java

**Comment by meteorcloudy** [comment]
@jvishnuvardhan Can someone from TensorFlow team look into this problem? I'm a bit occupied by my own work.

**Comment by DJSpyman** [comment]
Same here with 1.13.1 tried to narrow it down with procmon.exe because I also have a working developer machine, and a non working deploy device. Unluckily I could not narrow it down. By the Looks of it all dependant dlls were found.

**Comment by DJSpyman** [comment]
Hello Folks... I could solve this puzzle (at least for my issue).
After digging deeper and deeper I found that my processer (on the Target System) does not support AVX, but the library was built with AVX support. Therefore the DLL did not work. The same happend when I built the DLL for myself. 
I fixed it Building the DLL myself. During pyhton configure.py I had to set the option /arch:atom otherwise it would assume the AVX capabilities of my buildmachine CPU!

You might also checkout here: https://github.com/fo40225/tensorflow-windows-wheel
